browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -119,8 +119,7 @@ bool BraveActionsContainer::ShouldAddAction(const std::string& id) const {
 
 bool BraveActionsContainer::ShouldAddBraveRewardsAction() const {
   const PrefService* prefs = browser_->profile()->GetPrefs();
-  return prefs->GetBoolean(brave_rewards::prefs::kBraveRewardsEnabled) ||
-         !prefs->GetBoolean(kHideBraveRewardsButton);
+  return !prefs->GetBoolean(kHideBraveRewardsButton);
```

Comment:
We're just getting confirmation on whether the original logic/code was actually deliberate as a UX/design choice, before we merge this PR.